src/spec/controllers/admin/timeslots_controller_spec.rb

```diff
@@ -20,6 +20,7 @@
       expect(response).to be_successful
       expect(assigns[:event]).to eq event
       expect(assigns[:timeslot]).to be_kind_of Timeslot
+      expect(assigns[:timeslot].starts_at).to eq Date.parse('2015-10-15').in_time_zone
```

Comment:
I was surprised to see the `.in_time_zone` bit since it appears the controller action is ambivalent to TZs.

Also, might be simple/quick enough to add the assertion for title.